YuncunPlayer.md

```diff
@@ -0,0 +1,71 @@
+# <YuncunPlayer>
+
+
+
+## (0) Project Name and Team Slack Handles
+
+*Our project name is YuncunPlayer,.i am  one of filecoin miner *
+
+
+## (1) Brief Project Description
+
+This is an automatic voice sharing platform for users to upload their own songs, bedtime stories and other sounds to the Filecoin network, which can only be viewed by authorized users.
+
+
+## (2) Link to Project UI
+
+*This is our project UI link  http://www.binleo.com/
+THis is our project code link https://github.com/AlexY0905/YuncunHorizon-webui.git
+
+## (3) What does your application/UI do?
+
+Users share their own voices, even imitations of shows and musical covers，We share data through our platform
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*Mp3
```

Comment:
It appears that this application will be using user-uploaded data stored as MP3 and not a curated dataset.